src/main/java/raisetech/StudentManagement/data/Studentscourses.java

```diff
@@ -0,0 +1,18 @@
+package raisetech.StudentManagement.data;
+
+import java.time.LocalDateTime;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+
+public class Studentscourses {
```

Comment:
キャメルケースだと以下のようにするのが適切ですね。
```suggestion
public class StudentsCourses {
```